Auth-Page/login.css

```diff
@@ -0,0 +1,246 @@
+/* config.css */
+
+/* helpers/align.css */
+
+.align {
+    -webkit-box-align: center;
+        -ms-flex-align: center;
+            align-items: center;
+    display: -webkit-box;
+    display: -ms-flexbox;
+    display: flex;
+    -webkit-box-orient: vertical;
+    -webkit-box-direction: normal;
+        -ms-flex-direction: column;
+            flex-direction: column;
+    -webkit-box-pack: center;
+        -ms-flex-pack: center;
+            justify-content: center;
+  }
+img{
+    width: 65%;
+    margin-left: 2.8rem;
+}
+  .hero-logo{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    cursor: pointer;
+    margin-right: 3rem;
+    margin-bottom: 2rem;
+ }
+ .logo-mg{
+     width: 35%;
+     margin-right: -1rem;
+ }
+ .bg-color{
+     color: azure;
+     font-size: 1rem;
```

Comment:
you can declare variable names for colors

Reply:
ok sure